### ISSUE TITLE
Some users seeing old versions of TEAMMATES?

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#12367]
A user sent me this screenshot today:
![image](https://user-images.githubusercontent.com/1673303/231340115-fd41b149-07b7-461a-95d1-04e5c51ee944.png)

Based on the button color, seems like she is accessing a very old version of TEAMMATES. Is there anything we can do to prevent stale versions being accessed?

**Comment by wkurniawan07** [comment]
Not really. A large part of this is up to user's browser. Also, this has been known for a long while. Querying the maintainer logs would reveal that there is even a user whose frontend is stuck at V8.9.0 (> 1 year old) for whatever reason. The overwhelming majority of users are up-to-date or minus one version, so this is really not a big issue.

That said, one thing we can do is to check the expected version with the backend (need API for this) and force a hard refresh on mismatch, but this (1) again depends on browser, and (2) will only apply to new versions, whose users are in fact people who do not face the problem.

**Comment by anjum-edge** [comment]
Update the links in the ahrefs part to get it done
